simplivity/resources/policies.py

```diff
@@ -86,7 +86,7 @@ def create(self, name, flags=None, timeout=-1):
         """
         data = {"name": name}
 
-        affected_object = self._client.do_post(URL, data, timeout, flags)[0]
+        affected_object = self._client.do_post(URL, data, timeout, None, flags)[0]
```

Comment:
Since this change is coming in to fix an issue that was discovered, we should have a unit test that validates the fix.

Reply:
Added unit testcase with flags.